main.go

```diff
@@ -530,11 +544,38 @@ func canVote(groups []string) bool {
 		}
 	}
 
+	passesGatekeep := true
+	// check if the user passes gatekeep
+	if gatekeepEnforcedPoll {
+		endpointURL := CONDITIONAL_GATEKEEP_URL + username
+		req, err := http.NewRequest("GET", endpointURL, nil)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
Doesn't this `log.Fatal` terminate execution? There are a couple places where we use a custom `logger`. Check out [here](https://github.com/ComputerScienceHouse/vote/blob/main/database/database.go#L26) and [here](https://github.com/ComputerScienceHouse/vote/blob/main/logging/logger.go).